core/common/src/main/java/alluxio/conf/Reconfigurable.java

```diff
@@ -11,13 +11,16 @@
 
 package alluxio.conf;
 
+import java.util.Map;
+
 /**
  * Reconfigurable listener.
  */
 public interface Reconfigurable {
 
   /**
    * When the property changed, this function will be invoked.
+   * @param changedProperties the changed properties
    */
-  void update();
+  void update(Map<PropertyKey, Object> changedProperties);
```

Comment:
Keep the `update()` method which tells the listener that it needs to figure out the set of changed properties.

Reply:
done